jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -206,6 +206,10 @@ class DeploySamApplicationValidator {
         if (name.length > MAX_STACK_NAME_LENGTH) {
             return message("serverless.application.deploy.validation.new.stack.name.too.long", MAX_STACK_NAME_LENGTH)
         }
+        // Check if the new stack name is same as an existing stack name
+        if (existingNames.contains(name)) {
```

Comment:
Using the `in` operator is probably more idiomatic:

`if (name in existingNames) { .. }`

Reply:
Done.